apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { render, Result } from 'node-sass';
+import postcss from 'postcss';
+import cssModules from 'postcss-modules';
+import { StringValuesTypingsGenerator, IStringValueTypings } from '@rushstack/typings-generator';
+import { LegacyAdapters } from '@rushstack/node-core-library';
+
+/**
+ * @public
+ */
+export interface ISassConfiguration {
+  /**
+   * Source code root directory.
+   * Defaults to "src/".
+   */
+  srcFolder?: string;
+
+  /**
+   * Output directory for generated Sass typings.
+   * Defaults to "temp/sass-ts/".
+   */
+  generatedTsFolder?: string;
+
+  /**
+   * Determines if export values are wrapped in a default property, or not.
+   * Defaults to true.
+   */
+  exportAsDefault?: boolean;
+
+  /**
+   * The name of the export interface.
+   * Defaults to "IExportStyles".
+   */
+  exportAsInterfaceName?: string;
```

Comment:
Would anyone ever need to customize this?

Reply:
No one would ever need to, so I can remove this customization.